src/openrct2/paint/Supports.cpp

```diff
@@ -230,37 +246,39 @@ struct unk_supports_desc {
 
 /* 0x0097B23C */
 static constexpr const unk_supports_desc byte_97B23C[] = {
+    {{{0,  0,  0}, {1,  1,  8}},  0, 1}, // Flat to gentle
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
+    {{{0,  0,  0}, {1,  1,  8}},  0, 1}, // Gentle to flat
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
+    {{{0,  0,  0}, {1,  1,  8}},  0, 1}, // Gentle slope
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
     {{{0,  0,  0}, {1,  1,  8}},  0, 1},
-    {{{0,  0,  0}, {1,  1,  8}},  0, 1},
```

Comment:
I am not too sure about this. We're deviating from vanilla here, and I'm not sure why, or whether it will cause unforeseen problems with supports drawing.

Reply:
The behaviour of the existing supports should not have been modified. I'll look over the file again, I might have changed something accidentally.

Reply:
@Gymnasiast Can you clarify exactly what you think deviates from vanilla? I don't see any changes to the content of that array except for the extra entries I added.

Reply:
You inserted several lines in-between.

Reply:
Where? I compared the original version currently on develop and the version from this branch side by side and can't find any changes to the existing lines, other than adding some comments to mark which support each line belongs to (although I could remove these if you want).

Reply:
I have now checked locally. You are correct, it's just the way the diff is presented.